frontend/javascripts/oxalis/controller/scene_controller.ts

```diff
@@ -474,12 +475,13 @@ class SceneController {
     this.datasetBoundingBox.setVisibility(id !== OrthoViews.TDView || tdViewDisplayDatasetBorders);
     this.datasetBoundingBox.updateForCam(id);
     this.userBoundingBoxes.forEach((bbCube) => bbCube.updateForCam(id));
-    this.layerBoundingBoxes.forEach((bbCube) => {
-      const visible = id !== OrthoViews.TDView || tdViewDisplayLayerBorders;
+    const layerNameToIsDisabled = getLayerNameToIsDisabled(Store.getState().datasetConfiguration);
+    Object.keys(this.layerBoundingBoxes).forEach((layerName) => {
+      const bbCube = this.layerBoundingBoxes[layerName];
+      const visible =
+        id === OrthoViews.TDView && tdViewDisplayLayerBorders && !layerNameToIsDisabled[layerName];
```

Comment:
Yeah, I think, I went that route because the new layer bboxes are yellow now (to have at least a bit of a distinguishing characteristic) and if the layer bbox == dataset bbox, the borders would be yellow in the data viewports which felt a bit dominant.